src/content/service.js

```diff
@@ -78,7 +78,7 @@ module.exports = async ({ botfile, projectLocation, logger }) => {
     }
   }
 
-  const listCategoryItems = async (categoryId, from = 0, count = 50, searchTerm) => {
+  const listCategoryItems = async (categoryId, { from = 0, count = 50, searchTerm, orderBy = ['created_on'] }) => {
```

Comment:
if you put default arg `(categoryId, { from = 0, count = 50, searchTerm, orderBy = ['created_on'] } = {})` you won't need the passing empty options change below

Reply:
Good point, fixed that. )